webpack.config.js

```diff
@@ -17,7 +17,7 @@ function deepCopy (obj) {
 };
 
 const CopyWebPackFiles = [
-  "ISSUE_TEMPLATE.md",
+  ".github/ISSUE_TEMPLATE.md",
```

Comment:
Thanks. Now Firenvim builds correctly but the test that checks that Firenvim correctly autofills the template is failing. Could you fix that too? :)

Reply:
I guess webpack puts it into a subdirectory?

I am wondering whether we really have to match the template exactly and not simply replace the appropriate lines. This could also be useful in other contexts actually ^^

Reply:
No, I think it's just a matter of updating this path:

https://github.com/glacambre/firenvim/blob/565061ec6d437c5ce90485e2d99cd8bedab1cc0b/tests/_common.ts#L918

The test generates a page that looks like the github new issue page, along with the issue template. The test is complaining about not finding the template :).